governance/addin-mock/program/src/processor.rs

```diff
@@ -57,30 +66,23 @@ pub fn process_setup_voter_weight_record(
 ) -> ProgramResult {
     let account_info_iter = &mut accounts.iter();
 
-    let governance_program_info = next_account_info(account_info_iter)?; // 0
-    let realm_info = next_account_info(account_info_iter)?; // 1
-    let governing_token_mint_info = next_account_info(account_info_iter)?; // 2
-    let token_owner_record_info = next_account_info(account_info_iter)?; // 3
-    let voter_weight_record_info = next_account_info(account_info_iter)?; // 4
-    let payer_info = next_account_info(account_info_iter)?; // 5
-    let system_info = next_account_info(account_info_iter)?; // 6
-
-    let token_owner_record_data = get_token_owner_record_data_for_realm_and_governing_mint(
-        governance_program_info.key,
-        token_owner_record_info,
-        realm_info.key,
-        governing_token_mint_info.key,
-    )?;
```

Comment:
I removed this to break the circular dependency because `get_token_owner_record_data_for_realm_and_governing_mint` and `TokenOwnerRecord` are defined in` spl-governnace`. However it's also better for the mock  because it allows me to mock behaviour of badly implemented `addin` for test purposes.

I'm planning to create a library with `addin` implementations and also  a template project to make it simple for developers to create new ones